examples/src/armory/examples/object_detection/datasets/visdrone.py

```diff
@@ -0,0 +1,219 @@
+"""Utilities to load the VisDrone 2019 dataset."""
+
+import csv
+import io
+from pathlib import Path
+from pprint import pprint
+from typing import Any, Dict, Iterator, List, Tuple
+
+import albumentations as A
+import albumentations.pytorch
+import datasets
+import numpy as np
+
+import armory.data
+import armory.dataset
+
+
+def create_dataloader(
+    dataset: datasets.Dataset, max_size: int, **kwargs
+) -> armory.dataset.ObjectDetectionDataLoader:
+    """
+    Create an Armory object detection dataloader for the given VisDrone2019 dataset split.
+
+    Args:
+        dataset: VisDrone2019 dataset split
+        max_size: Maximum image size to which to resize and pad image samples
+        **kwargs: Additional keyword arguments to pass to the dataloader constructor
+
+    Return:
+        Armory object detection dataloader
+    """
+    resize = A.Compose(
+        [
+            A.LongestMaxSize(max_size=max_size),
+            A.PadIfNeeded(
```

Comment:
It may not be necessary to pad all input images to the max size, some models take a list of images as input (e.g. Faster-RCNN).